pkg/util/crypt/crypt_dev.go

```diff
@@ -299,6 +300,19 @@ func (crypt *Device) Open(key []byte, path string) (string, error) {
 
 			return "", fmt.Errorf("cryptsetup open failed: %s: %v", string(out), err)
 		}
+
+		for attempt := 0; true; attempt++ {
+			_, err := os.Stat("/dev/mapper/" + nextCrypt)
+			if err == nil {
```

Comment:
In checks for the existence of a file/directory/device, it's usual to check if `os.IsNotExist(err)` is true, and fail immediately if the err is not an IsNotExist.

Reply:
Done.